data/mapper/config/config.go

```diff
@@ -0,0 +1,32 @@
+package config
+
+import (
+	"os"
+	"strconv"
+)
+
+const (
+	EnvMappingIgnoreError        = "FLOGO_MAPPING_IGNORE_ERRORS"
+	EnvMappingIgnoreErrorDefault = false
+
+	EnvMappingSkipMissing        = "FLOGO_MAPPING_SKIP_MISSING"
+	EnvMappingSkipMissingDefault = false
+)
+
+func IsMappingIgnoreError() bool {
```

Comment:
Consider renaming to IsMappingIgnoreErrorsOn() or IsMappingIgnoreErrorsEnabled()